PocketKit/Sources/PocketKit/Home/CompactHomeCoordinator.swift

```diff
@@ -269,7 +303,11 @@ class CompactHomeCoordinator: NSObject {
 
     private func present(alert: PocketAlert?) {
         guard !isResetting, let alert = alert else { return }
-        viewController.present(UIAlertController(alert), animated: !isResetting)
+        guard let presentedVC = viewController.presentedViewController else {
```

Comment:
Can you briefly explain under what scenario this was causing issue?

Reply:
When this method is called,
viewController is UINavigationController and showing an alert on that is fine but if user is at Home. But when user open a Webview, then alert won't show since it is presenting on navigation not on Webview.

By getting presentedVC, we get what is already preseting (Webview in this case) and show alert on that.
presentedVC = viewController.presentedViewController


User cases for showing alert:
> When the alert shows on Home?
If user tap on 3 dots (more options) for any saved items under Recent Saves on home screen, user can choose options to delete the saved item which further shows an alert.

> When the alert shows on Webview?
If user open saved recommendation in Webview and click on share menu, user can choose delete from the list.

Reply:
@cyndichin This is reference to your comment for 'CompactSavesContainerCoordinator.swift' class and above is the brief reasoning on why we have this navigation logic.